integration-tests/bats/sql-client.bats

```diff
@@ -147,3 +147,15 @@ teardown() {
     [ $status -eq 0 ]
     [[ $output =~ " test_dashes " ]] || false
 }
+
+@test "sql-client: prints accurate query timing" {
+    skiponwindows "Missing dependencies"
```

Comment:
Are you sure this one won't run on windows? It looks pretty similar to other tests in this file that don't seem to be skipped on windows. Worth trying out if you haven't tested it yet.